app/controllers/application_controller.rb

```diff
@@ -1,2 +1,22 @@
 class ApplicationController < ActionController::Base
+  # [...]
+  before_action :authenticate_user!
+  include Pundit
+
+  # Pundit: white-list approach.
+  after_action :verify_authorized, except: :index, unless: :skip_pundit?
```

Comment:
don't think we want to exclude the index; because only the user will have access to the index of its lists